cmd/qat_plugin/README.md

```diff
@@ -187,6 +187,19 @@ For example, `qat.intel.com/generic: <number of devices>` for a container reques
 
 For a DPDK-based workload, you may need to add hugepage request and limit.
 
+##### NOTES
```

Comment:
we don't need a separate note for this. Can we just add in parenthesis on line 186 something like: "(in case for 4th Gen Xeon and later, use `cy` or `dc` instead of `generic` for crypto and compression functionality, respectively)".